demos/CompositeUI-MVC/Divergent.ITOps.ViewModelComposition/DynamicViewModel.cs

```diff
@@ -77,9 +84,28 @@ public override bool TryInvokeMember(InvokeMemberBinder binder, object[] args, o
             return false;
         }
 
+        public override IEnumerable<string> GetDynamicMemberNames()
+        {
+            return base.GetDynamicMemberNames()
+                .Union(properties.Keys)
+                .Union(new[] { "RaiseEventAsync", "RaiseEvent" });
```

Comment:
In a real app, would this be a hot path? Would using `yield return` be better than relying upon LINQ?